params.go

```diff
@@ -70,6 +70,7 @@ func (f Filters) AppendTo(body *form.Values, keyParts []string) {
 // of List iterators. The Count property is only populated if the
 // total_count include option is passed in (see tests for example).
 type ListMeta struct {
```

Comment:
Damn I kinda wish we renamed that one to `StripeList` instead.

Reply:
I think `ListMeta` kind of makes sense because the `Data` parameter of a list is actually on the leaf struct, which makes `ListMeta` not a complete list. e.g.

``` go
type CouponList struct {
	ListMeta
	Data []*Coupon `json:"data"`
}
```

One alternative I thought about here was adding `APIResource` to each individual list instead, which in some ways might be a bit more logical. e.g.

``` go
type CouponList struct {
	APIResource
	ListMeta
	Data []*Coupon `json:"data"`
}
```

Don't think it'd make too much practical difference overall . Thoughts?

Reply:
Actually, you know what, I think I'm going to change this. Feels cleaner/more consistent to me.